client/wordpress-com.js

```diff
@@ -67,6 +67,13 @@ sections = [
 		secondary: true
 	},
 	{
+		name: 'chat',
```

Comment:
We should move this outside of the `sections` literal and wrap inside an `isEnabled` check further down the page.